app/assets/javascripts/discourse/app/components/sidebar/user/section-link.js

```diff
@@ -23,7 +24,23 @@ export default class SectionLink {
 
   @bind
   didStartDrag(e) {
```

Comment:
nit, but it's easier to search for these if they are called `event` rather than just `e`